app/views/items/deleted.html.erb

```diff
@@ -0,0 +1,47 @@
+<% content_for :title, "Inventory" %>
+
+<% content_for :sidebar do %>
+  <%= render "common/category_nav" %>
+<% end %>
+
+<% content_for :content_size, "col-sm-9 col-md-10" %>
+
+<% content_for :content do %>
+  <% if @items.empty? %>
+    <h4 class="button-height">
+      <%= @category %>
+    </h4>
+    <hr />
+    <p class="text-danger">
+      There are no deleted items in this category.
+    </p>
+  <% else %>
+    <h4 class="button-height">
+      <%= @category %>
+    </h4>
+
+    <table class="table table-hover table-striped data-table no-paging">
+      <thead>
+        <tr>
+          <th class="col-xs-4">Category</th>
+          <th class="col-xs-4">Description</th>
+          <th class="text-center">SKU</th>
+          <th class="text-center">Actions</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @items.each do |item| %>
+          <tr>
+            <td class="col-xs-4"><%= item.category.description %>
+            <td class="col-xs-4"><%= item.description %></td>
+            <td class="text-center"><%= item.sku %></td>
+            <td class="text-center">
+              <%= link_to "Restore", restore_item_path(item), class: "btn-sm btn-success", data: confirm(title: "Restoring Item: #{item.description}") %>
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  <% end %>
+<% end %>
```

Comment:
Is this view mostly the same as the index? It might be nice to share common code